### ISSUE TITLE
App crash in release mode

### ISSUE THREAD

**Issue by tsvietOK** [duke7553/files-uwp#720]
**Describe the bug**
Sometimes app crash right after launch.

**To Reproduce**
1. Build app in release mode and deploy
2. Start app
3. If it did not crash, restart app

**Expected behavior**
Expected normal start without crash

**Desktop (please complete the following information):**
 - OS: [e.g. Windows 10 1909 18363.815]
 - App version: [v0.7.9]

**Additional context**
Built from develop branch.

**Comment by yaichenbaum** [comment]
I wonder if this is related to the changes @ADeltaX made...

**Comment by tsvietOK** [comment]
@yaichenbaum Just reverted his changes and it is still crash

**Comment by tsvietOK** [comment]
@yaichenbaum As i understand, that happen because of my changes about theme change

**Comment by yaichenbaum** [comment]
Oh, I did notice you switched to a different method for the get\set.

**Comment by tsvietOK** [comment]
What do you mean?

**Comment by yaichenbaum** [comment]
Before your change, the setting for acrylic was a simple get and set, this would automatically save the setting as well saving us from needing to detect it when launching the app. An example of how it should be is,
```
public bool AcrylicSidebar
{
     get => Get(false);
     set => Set(value)
}
```

**Comment by tsvietOK** [comment]
I remember this, but this implementation was buggy for toggle switches, to enable or disable it you had to click it twice. So i reworked it and it work good in debug mode.

**Comment by yaichenbaum** [comment]
We need to figure out why this implementation causes that issue, it's worth it to use the implementation I shared above since it is more compact. I have used a similar implementation for settings in others apps and it worked fine, so I must of missed something when copying it over.

**Comment by tsvietOK** [comment]
Good example, preference page, onedrive toggle working good, to enable or disable next toggle(show dialog...) you have to click twice, but if you click only once and reload settings page, toggle will change  state. But, anyway, i don't think get and set of acrylic parameter can cause app crash.

**Comment by ADeltaX** [comment]
> I wonder if this is related to the changes @ADeltaX made...

My code is 200% stable 😂😂😂

Anyway, jokes aside, I've had the same issue before my commit, and only in Release mode, so...
(PS: it happened on 0.7.7 store version for some reason but it's more random and rare)

**Comment by yaichenbaum** [comment]
I canceled the rollout of the v0.7.9 release. I am not sure what's going on but I would appreciate if someone can figure it out.

**Comment by tsvietOK** [comment]
@yaichenbaum We can revert my changes until problem will not be fixed. Currently i really don't understand what's wrong with my patch in release mode. Any help appreciated.

**Comment by yaichenbaum** [comment]
@tsvietOK Did you get an exception when you tried running the app in release mode?

**Comment by tsvietOK** [comment]
@yaichenbaum No, i can't catch any exception, app just crash right after start(Sometimes it starts fine).

**Comment by tsvietOK** [comment]
@yaichenbaum Some notice, if comment [line](https://github.com/tsvietOK/files-uwp/blob/36e1853cd6ab068b28d1c1417e31aeca8e295140/Files/App.xaml.cs#L236) app starts fine(of course theme change in this case is not working)

**Comment by yaichenbaum** [comment]
"Unhandled exception at 0x00007FFA571232E0 (Windows.UI.Xaml.dll) in Files.exe: 0xC000027B: An application-internal exception has occurred (parameters: 0x000001ECFDD83BA0, 0x0000000000000002)."

**Comment by tsvietOK** [comment]
@yaichenbaum Sorry, i don't know what to do with this info:(

**Comment by yaichenbaum** [comment]
@tsvietOK I tried commenting out the line you linked and I still get the issue.

**Comment by tsvietOK** [comment]
@yaichenbaum There is another [line](https://github.com/tsvietOK/files-uwp/blob/36e1853cd6ab068b28d1c1417e31aeca8e295140/Files/App.xaml.cs#L255).

**Comment by tsvietOK** [comment]
@yaichenbaum problem somewhere in lines 
https://github.com/tsvietOK/files-uwp/blob/36e1853cd6ab068b28d1c1417e31aeca8e295140/Files/Helpers/ThemeHelper.cs#L113
https://github.com/tsvietOK/files-uwp/blob/36e1853cd6ab068b28d1c1417e31aeca8e295140/Files/Helpers/ThemeHelper.cs#L119
https://github.com/tsvietOK/files-uwp/blob/36e1853cd6ab068b28d1c1417e31aeca8e295140/Files/Helpers/ThemeHelper.cs#L125
If comment them, it works fine

**Comment by tsvietOK** [comment]
@yaichenbaum The problem is in var TintLuminosityOpacity in AppTheme. For some reason setting a value to this var in methods SetDefaultTheme(), SetLightTheme() and SetDarkTheme() causes app crash.

**Comment by yaichenbaum** [comment]
It turns out this issue goes back pretty far, it shouldn't be an issue for the store build.

**Comment by tsvietOK** [comment]
@yaichenbaum We can keep this PR open until we will be sure that there is no issue for the store build

**Comment by tsvietOK** [comment]
@yaichenbaum Bug in v0.7.9 still exists

**Comment by yaichenbaum** [comment]
@tsvietOK Even if you open the app a second time?

**Comment by tsvietOK** [comment]
@yaichenbaum After reinstall and clear data first and second start - crash, third start successful

**Comment by yaichenbaum** [comment]
That is fine, does it work to create a new tab?

**Comment by tsvietOK** [comment]
@yaichenbaum It crash when opening new tab

**Comment by tsvietOK** [comment]
@yaichenbaum My PR should fix that

**Comment by yaichenbaum** [comment]
@tsvietOK Thank you for working on this issue, the fix is now out in production.
